apps/admin-x-activitypub/src/components/layout/Sidebar/Sidebar.tsx

```
@@ -40,30 +41,24 @@ const Sidebar: React.FC = () => {
-                                        </span>
-                                    </div>
-                                );
-                            }
-                            return (<></>);
-                        })}
-                    </div>
+                                    </span>
+                                </div>
+                            );
+                        }
+                        return (<></>);
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)